Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -76,6 +76,30 @@ open class NavigationMapView: UIView {
         }
     }
     
+    /**
+     Controls whether the main route style layer and its casing disappears
+     as the user location puck travels over it. Defaults to `false`.
+     
+     Used in standalone `NavigationMapView` during navigation. If using `NavigationViewController` and `CarPlayNavigationViewController`
+     for active navigation, update `NavigationViewController.routeLineTracksTraversal` and `CarPlayNavigationViewController.routeLineTracksTraversal` instead.
+     
+     If `true`, the part of the route that has been traversed will be
+     rendered with full transparency, to give the illusion of a
+     disappearing route. To customize the color that appears on the
+     traversed section of a route, override the `traversedRouteColor` property
+     for the `NavigationMapView.appearance()`. If `false`, the whole route will be shown without traversed
+     part disappearing effect.
+     
+     To update the route line during active navigation, add observers for `routeControllerDidRefreshRoute`,
```

Comment:
I think that we usually mention these notifications as `Notification.Name.routeControllerDidRefreshRoute`.

Reply:
Thanks for the suggestion, this part has been renamed and updated.